lando/README.md

```diff
@@ -34,22 +34,41 @@ _Prerequisite: Make sure you have added your SSH key in Acquia cloud, and that i
 - 'username' => 'drupal',
 - 'password' => 'drupal',
 - 'host' => 'database',
-8. Run `lando blt drupal:sync --site=default --sync-files` to pull down a copy of the database and files for the default multisite.
+8. Run `lando blt drupal:sync --site=default --sync-files` to pull down a copy of the database and files for the default multisite. If you want to get the custom views built out on on the database you are pulling from, you can also use: `lando blt drupal:sync --site=default --sync-files --partial`.
```

Comment:
Do you run this command in addition to the first or in place of it?

Reply:
Yeah it's either / or, but honestly, pulling down the partials might be more useful. I'll try to clarify this!